test.js

```diff
@@ -1,77 +1,33 @@
 import { equal } from 'assert';
 import mdast from 'mdast';
-import typographer from './';
+import base from 'typographic-base';
+import typographer from './index';
 
 const t = (text, options) =>
   mdast().use(typographer, options).process(text).trim();
```

Comment:
one more point came to me yesterday, separation of concerns of plugins and options will keep it consistent with textr itself and keep it clear from conflict error